### ISSUE TITLE
Printable version

### ISSUE THREAD

**Issue by darrell** [pdxosgeo/foss4g2014-wordpress#33]
Is it worth creating a printable version of program?

**Comment by mapsam** [comment]
Is there going to be one handed out at the conference?

**Comment by darrell** [comment]
Yes, but I had someone ask about it, and it kind of does make sense for those who like to plan ahead, though it’s not a _huge_ priority.

d.
